examples/glue_benchmark/glue.py

```diff
@@ -195,41 +225,55 @@ def preprocess_fn(feature, label):
     val_ds = preprocess_data(preprocess_fn, val_ds)
     test_ds = preprocess_data(preprocess_fn, test_ds)
 
-    loss = keras.losses.SparseCategoricalCrossentropy(from_logits=True)
-    metrics = [keras.metrics.SparseCategoricalAccuracy()]
-    if FLAGS.task_name == "stsb":
-        num_classes = 1
-        loss = keras.losses.MeanSquaredError()
-        metrics = [keras.losses.MeanSquaredError()]
-    elif FLAGS.task_name in ("mnli", "mnli_mismatched", "mnli_matched", "ax"):
-        num_classes = 3
+    if FLAGS.load_finetuning_model:
+        finetuning_model = tf.keras.models.load_model(
+            FLAGS.load_finetuning_model
+        )
     else:
-        num_classes = 2
-
-    # ----- Custom code block starts -----
-    # Users should change this `BertClassifier` to your own classifier.
-    # Commonly the classifier is simply your model + several dense layers,
-    # please refer to "Make the Finetuning Model" section in README for
-    # detailed instructions.
-    finetuning_model = keras_nlp.models.BertClassifier(
-        backbone=bert_model,
-        num_classes=num_classes,
-    )
-    # ----- Custom code block ends -----
+        loss = keras.losses.SparseCategoricalCrossentropy(from_logits=True)
+        metrics = [keras.metrics.SparseCategoricalAccuracy()]
+        if FLAGS.task_name == "stsb":
+            num_classes = 1
+            loss = keras.losses.MeanSquaredError()
+            metrics = [keras.metrics.MeanSquaredError()]
+        elif FLAGS.task_name in (
+            "mnli",
+            "mnli_mismatched",
+            "mnli_matched",
+            "ax",
+        ):
+            num_classes = 3
+        else:
+            num_classes = 2
+
+        # ----- Custom code block starts -----
+        # Users should change this `BertClassifier` to your own classifier.
+        # Commonly the classifier is simply your model + several dense layers,
+        # please refer to "Make the Finetuning Model" section in README for
+        # detailed instructions.
+        finetuning_model = keras_nlp.models.BertClassifier(
+            backbone=bert_model,
+            num_classes=num_classes,
+        )
+        # ----- Custom code block ends -----
 
-    finetuning_model.compile(
-        optimizer=tf.keras.optimizers.Adam(FLAGS.learning_rate),
-        loss=loss,
-        metrics=metrics,
-    )
+        finetuning_model.compile(
+            optimizer=tf.keras.optimizers.Adam(FLAGS.learning_rate),
```

Comment:
Should we show AdamW here? It is honestly far more common if every transformer codebase I have seen, if might be the better default. (Just with default args).